lib/data/client-audits.js

```diff
@@ -16,7 +16,7 @@ const { zipPart } = require('../util/zip');
 const headers = [ 'event', 'node', 'start', 'end', 'latitude', 'longitude', 'accuracy', 'old-value', 'new-value' ];
 
 // used by parseClientAudits below.
-const parseOptions = { bom: true, trim: true, skipEmptyLines: true };
+const parseOptions = { bom: true, trim: true, skipEmptyLines: true, relax: true };
```

Comment:
Yasss, one little form that has been broken for me since I reproduced this issue (https://github.com/getodk/collect/issues/4473) now exports properly.